int-set/src/lib.rs

```diff
@@ -40,6 +40,13 @@ impl<T: Into<u32> + Copy> Default for IntSet<T> {
 }
 
 impl<T: Into<u32> + Copy> IntSet<T> {
+    // TODO(garretrieger): add additional functionality that the harfbuzz version has:
+    // - Iteration in reverse.
```

Comment:
There is the trait in `std` called [`DoubleEndedIterator`](https://doc.rust-lang.org/1.78.0/std/iter/trait.DoubleEndedIterator.html); if you implement this, then you also get a `rev()` method, which reverses iteration order.

Reply:
ah nice, yes that looks like the way to go.

Reply:
I'll do this in a separate PR.